internal/testing/runchecks.sh

```diff
@@ -28,6 +28,11 @@ fi
 
 result=0
 
+if [[ "$TRAVIS_OS_NAME" == "windows" && "$TRAVIS_EVENT_TYPE" != "cron" ]]; then
```

Comment:
Could you make this `== "pullrequest"` or whatever to make it opt-out instead of opt-in?

Reply:
Done.